fish_prompt.fish

```diff
@@ -81,9 +81,8 @@ function fish_prompt
     end
 
     # Check if there is an upstream configured
-    set -l has_upstream (command git rev-parse --abbrev-ref @'{u}' ^/dev/null)
-
-    if test -n "$has_upstream"
+    command git rev-parse --abbrev-ref '@{upstream}' >/dev/null ^&1
+    if test $status -eq 0
       set -l git_status (command git rev-list --left-right --count HEAD...@'{u}' | sed "s/[[:blank:]]/ /" ^/dev/null)
```

Comment:
It's more of a personal habit than a must, but it's best to be consistent, so I will change this.